src/runtime/runtime_wasm_wasip2.go

```diff
@@ -0,0 +1,71 @@
+//go:build wasip2
+
+package runtime
+
+import (
+	"unsafe"
+
+	"internal/wasi/cli/v0.2.0/environment"
+	monotonicclock "internal/wasi/clocks/v0.2.0/monotonic-clock"
+)
+
+type timeUnit int64
+
+// libc constructors
+//
+//export __wasm_call_ctors
+func __wasm_call_ctors()
+
+//export wasi:cli/run@0.2.0#run
+func __wasi_cli_run_run() uint32 {
+	_start()
+	return 0
+}
+
+//export _start
+func _start() {
+	// These need to be initialized early so that the heap can be initialized.
+	heapStart = uintptr(unsafe.Pointer(&heapStartSymbol))
+	heapEnd = uintptr(wasm_memory_size(0) * wasmPageSize)
+	run()
+}
+
+// Read the command line arguments from WASI.
+// For example, they can be passed to a program with wasmtime like this:
+//
+//	wasmtime run ./program.wasm arg1 arg2
+func init() {
+	__wasm_call_ctors()
+}
```

Comment:
If `__wasm_call_ctors()` does nothing, the comment is misleading.

Reply:
`__wasm_call_ctors()` stuff removed.